src/frontend/admin/src/components/templates/accesses/list/AddUserAccess.spec.tsx

```diff
@@ -0,0 +1,31 @@
+import { render, screen } from "@testing-library/react";
+import { CourseAccesses } from "@/services/api/models/Course";
+import { TestingWrapper } from "@/components/testing/TestingWrapper";
+import { AddUserAccess } from "@/components/templates/accesses/list/AddUserAccess";
+
+describe("<AddUserAccess />", () => {
+  it("render component", async () => {
+    const allAccesses = Object.values(CourseAccesses);
+    const roles = allAccesses.map((item) => ({
+      label: item,
+      value: item,
+    }));
+
+    const onAdd = jest.fn();
+
+    render(
+      <AddUserAccess
+        allAccesses={roles}
+        defaultRole={roles[0].value}
+        onAdd={onAdd}
+      />,
+      {
+        wrapper: TestingWrapper,
+      },
+    );
+
+    await screen.findByRole("combobox");
+    screen.getByRole("button", { name: "Add" });
```

Comment:
Maybe a good idea to fire a click on this button then check onAdd has been called once with the proper arguments ?